src/main/cpp/RobotContainer.cpp

```diff
@@ -83,6 +99,14 @@ bool RobotContainer::InitActuators(ActuatorInterface *actuators_interface) {
     actuators_interface->right_motor_2->SetIdleMode(COAST);
     actuators_interface->right_motor_3->SetIdleMode(COAST);
 
+    actuators_interface->intake_position_motor = std::make_unique<rev::CANSparkMax>(INTAKE_POSITION_MOTOR, BRUSHLESS);
+    actuators_interface->intake_motor = std::make_unique<rev::CANSparkMax>(INTAKE_MOTOR, BRUSHLESS);
+    actuators_interface->indexer_motor = std::make_unique<rev::CANSparkMax>(INDEXER_MOTOR, BRUSHLESS);
+
+    actuators_interface->intake_position_motor->SetIdleMode(COAST);
+    actuators_interface->intake_motor->SetIdleMode(COAST);
```

Comment:
Were these coast in the java code? If so carry on. We probably should try brake in the future

Reply:
they were, yes. I was surprised too